packages/webview_flutter/webview_flutter_wkwebview/lib/src/web_kit/web_kit.dart

```diff
@@ -573,4 +603,18 @@ class WKWebView extends NSObject {
   Future<Object?> evaluateJavaScript(String javaScriptString) {
     throw UnimplementedError();
   }
+
+  /// The view’s background color.
+  ///
+  /// The default value is null, which results in a transparent background color.
+  ///
+  /// Sets [UIView.backgroundColor](https://developer.apple.com/documentation/uikit/uiview/1622591-backgroundcolor?language=objc).
+  Future<void> setBackgroundColor(Color? color) {
+    throw UnimplementedError();
+  }
+
+  /// Determines whether the view is opaque.
+  Future<void> setOpaque(bool opaque) {
+    throw UnimplementedError();
+  }
```

Comment:
Is it worth adding a UIView base class and `implements`-ing it at some point? (Not something we need to do in this PR certainly.)

Reply:
It only takes about 20 lines of code, so I went ahead and added `UIView`.